Sources/AppStoreConnectCLI/Services/Operations/CreateProfileOperation.swift

```diff
@@ -32,5 +32,13 @@ struct CreateProfileOperation: APIOperation {
             .map { $0.data }
             .eraseToAnyPublisher()
     }
+    
+    
```

Comment:
The added whitespace here is triggering `swiftlint` if you open the workspace instead of the package you should be able to see the warnings more easily.
```suggestion
```